crates/router/src/connector/bankofamerica/transformers.rs

```diff
@@ -97,8 +99,10 @@ pub struct CaptureOptions {
 }
 
 #[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
 pub struct PaymentInformation {
-    card: Card,
+    card: Option<Card>,
```

Comment:
Create enum for PaymentInformation, one for card and one for fluid data

Reply:
Done